app/views/layouts/ats/profile.html.slim

```diff
@@ -6,7 +6,7 @@
         = yield :header
 
       - if content_for?(:tab_list)
-        section#tab-list.row.mt-2
```

Comment:
fixed this problem, line moved left
![image](https://github.com/user-attachments/assets/7720936a-cf32-4b65-a6b5-01016227c901)